internal/io/memory/memory_test.go

```diff
@@ -83,7 +85,6 @@ func TestSharedInmemoryNode(t *testing.T) {
 				t.Errorf("result %s should be equal to %s", res, expected)
 			}
 			return
-		default:
```

Comment:
This is to return immediately if no receiving anything, otherwise the test will be blocked for error case.

Reply:
So instead, we need to `return` after `default`?

Reply:
This is a little tricky.  There are actually two receivers and only one sender. So we need to skip if no receive and try for the next time.

Reply:
I see, then I think we need to add a `time.Sleep(1 * time.Millisecond)` to prevent the loop spin.

Reply:
OK, I will bypass this check in the lint PR.